lib/noop.js

```diff
@@ -0,0 +1 @@
+module.exports = exports = function(){};
```

Comment:
There is no reason for this file to exist. If you need an empty function in a test or as a base case for an injected dependency, you should inline a function where it is needed.

Reply:
How to test the function is a noop. See `master_spec.js` line 155